slack-notify/README.md

```diff
@@ -0,0 +1,56 @@
+# slack-notify
```

Comment:
Great with a flexible Slack action. Maybe it would be useful to add some information about when to use this vs. the older `slack-message` action? And perhaps also update the `slack-message` action to mention this new action?